lib/OrdinaryDiffEqLowOrderRK/src/low_order_rk_caches.jl

```diff
@@ -1535,3 +1485,27 @@ function alg_cache(alg::Alshina6, u, rate_prototype, ::Type{uEltypeNoUnits},
     Alshina6Cache(u, uprev, k1, k2, k3, k4, k5, k6, k7, tmp, tab, alg.stage_limiter!,
         alg.step_limiter!, alg.thread)
 end
+
+@cache struct FunctionMapCache{uType, rateType} <: OrdinaryDiffEqMutableCache
```

Comment:
FunctionMap is separate from the LowOrderRK methods because it acts on discrete problems instead of ODEs.

Reply:
Should I make it a separate library for this one ?? similar to Tsit ??

Reply:
Yes